lib/GridItem.jsx

```diff
@@ -1,18 +1,16 @@
 // @flow
-import {default as React, Component} from 'react';
-import {perc, setTransform} from './utils';
+import React from 'react';
 import Draggable from 'react-draggable';
 import {Resizable} from 'react-resizable';
-import assign from 'object-assign';
+import {perc, setTransform} from './utils';
 
-import type {Position, Size} from './utils';
+import type {Position} from './utils';
 
 /**
  * An individual item within a ReactGridLayout.
  */
-export default class GridItem extends Component {
-
-  static propTypes = {
+const GridItem = React.createClass({
```

Comment:
I could gather all needed changes in an updated PR within the next days.
Thanks for your advise on ES6 classes I think you have a very valid point. I will revert the changes by tomorrow.